shared/troubleshooting/jetson-tx2.md

```diff
@@ -18,9 +18,8 @@ Starting with L4T 32.1 the L4T partition layout has changed as well as the firmw
 
 In the case of a device that will not boot because of an interrupted update procedure, it will need to be re-flashed. This can be done by [downloading the L4T Driver Package][l4t-download] (BSP), unpacking it, putting the device in recovery mode and executing `sudo ./flash.sh jetson-tx2 mmcblk0p1`. Once the L4T partitions are finished writing, the device can be booted using the appropriate balenaOS flasher SD-CARD image for the targeted L4T version.
 
+
+[l4t]:https://developer.nvidia.com/embedded/linux-tegra
 [l4t-download]:https://developer.nvidia.com/embedded/linux-tegra-archive
 [meta-tegra]:https://github.com/OE4T/meta-tegra
 [self-service-update]:/reference/OS/updates/self-service/
-[tegra-binaries-version]:https://github.com/balena-os/balena-jetson/tree/master/layers/meta-balena-jetson/recipes-bsp/tegra-binaries
-[jetson-flash]:https://github.com/balena-os/jetson-flash
-[hardware section]:/learn/develop/hardware/i2c-and-spi/#jetson-devices
```

Comment:
Does this section get included automatically for the TX2? I see it for each Nano but not for the TX2 and TX2 NX so making sure we have it:
```

### Custom device tree support

Loading of custom device trees (DT) in balenaOS is supported only by the Jetson Nano and Jetson TX2 family of devices, which have u-boot support. The complete list of devices supporting custom DT and how to apply them is available in the [hardware section][hardware section].

[hardware section]:/learn/develop/hardware/i2c-and-spi/#jetson-devices
```

Reply:
Added

Reply:
Perfect! Thanks @vipulgupta2048 !